src/software/jetson_nano/services/motor.cpp

```diff
@@ -78,6 +107,9 @@ MotorService::MotorService(const RobotConstants_t& robot_constants,
     OPEN_SPI_FILE_DESCRIPTOR(back_left, BACK_LEFT_MOTOR_CHIP_SELECT)
     OPEN_SPI_FILE_DESCRIPTOR(back_right, BACK_RIGHT_MOTOR_CHIP_SELECT)
     OPEN_SPI_FILE_DESCRIPTOR(dribbler, DRIBBLER_MOTOR_CHIP_SELECT)
+
+    // Make this instance available to the static functions above
+    g_motor_service = this;
```

Comment:
yikes :grimacing:

Reply:
yeah :/ not sure if there is a better way to call the motor service class functions from externed c functions in the trinamic api